src/sensor.cpp

```diff
@@ -628,6 +628,18 @@ void log_callback_end( uint32_t fps,
         raise_on_before_streaming_changes(true); //Required to be just before actual start allow recording to work
         _source.set_callback(callback);
         _is_streaming = true;
+
+        std::string device_name = this->get_device().shared_from_this()->get_info(RS2_CAMERA_INFO_NAME);
+        auto active_streams = get_active_streams();
+        for (auto profile : active_streams)
+        {
+            auto stream_type = rs2_stream_to_string(profile->get_stream_type());
+            std::string timer_name = librealsense::aus_build_system_timer_name(stream_type, device_name);
```

Comment:
Do we sometimes create a timer and doesn't start it a line after?
What happened if this is called twice?
create 2 timers with the same name and start them?